sql/connect/common/src/main/protobuf/spark/connect/pipelines.proto

```diff
@@ -154,9 +154,11 @@ message PipelineCommandResult {
     optional string dataflow_graph_id = 1;
   }
   message DefineDatasetResult {
+    // A quoted identifier string representing fully resolved name of the dataset
     optional string resolved_data_name = 1;
   }
   message DefineFlowResult {
+    // A quoted identifier string representing fully resolved name of the flow
```

Comment:
let's just say the `resolved name of the flow` and don't expose the implementation detail in the API doc, as implementation can always be changed